regolith/schemas.py

```diff
@@ -3753,38 +3769,49 @@
                     "name": {"description": "what is the deliverable of milestone",
                              "required": False,
                              "type": "string"},
-                    "notes": {"description": "small, non-deliverable to-do's",
+                    "notes": {"description": "any notes about the milestone and/or "
+                                             "small, non-deliverable to-dos to reach the milestone",
+                              "required": False,
+                              "type": "list"},
+                    "progress": {"description": "text description of progress and observations, "
```

Comment:
the schema is not quite right here.  It should be a list of dicts and the dict has to have its own subschema

Reply:
This needs fixing.  Look elsewhere in the schema for a model